ATT-1: Enable rotation of image attachments, adding the javascript cod…

Wandji69 opened the pull request:
…e and enabling the java code to call 'rotate instruction
https://issues.openmrs.org/browse/ATT-1

HerbertYiga commented:
did you see the failing ci build tests?

Wandji69 commented:
I did and i am trying to get my head around the code base to see how i
could write the test cases

On Sun, Apr 28, 2019 at 4:05 PM HerbertYiga <notifications@github.com>
wrote:

> did you see the failing ci build tests?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openmrs/openmrs-module-attachments/pull/28#issuecomment-487387526>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHHQANYYLHCGEVNN2SYAOPLPSW4LTANCNFSM4HAETB7Q>
> .
>

mks-d commented:
@Wandji69 this should probably be a draft PR while the ticket is still in design.

This will require to modify the UI as well, so that the thumbnail 'edit mode' shows a rotation icon/button, only for attachments of the image family.

Commit:
ATT-1: Making images rotable in edit mode

Commit:
pulling updates from master
Merge branch 'master' of https://github.com/openmrs/openmrs-module-attachments into ATT-1

Commit:
fix failing text

Wandji69 commented:
I have implemented the rotation of images and I have issues on the test part please can some help me look into this.
